src/System.Text.Json/tests/Serialization/ExceptionTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public static partial class ExceptionTests
```

Comment:
What's the plan for serialization (i.e. writing) tests?

Reply:
I have another PR out that handles one exception case. Then there will be at least one more exception to prevent recursion by setting\using MaxDepth.